### PR TITLE
Editorial: Better explanation of the Global extended attribute.

### PR DESCRIPTION
**Pull request by Ms2ger**
CC #748.

***
<a href="https://pr-preview.s3.amazonaws.com/heycam/webidl/pull/749.html" title="Last updated on Apr 29, 2020, 3:33 PM UTC (351e093)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/heycam/webidl/749/350fb2a...351e093.html" title="Last updated on Apr 29, 2020, 3:33 PM UTC (351e093)">Diff</a>

### CODE REVIEW

**Comment by Ms2ger** [comment]
@tabatkins this doesn't add an example yet, but I hope it already helps to clarify how `[Global]` is supposed to work.

File: index.bs [hunk 1]
```diff
@@ -8673,12 +8673,32 @@ for the specific requirements that the use of
...
+     :: « the given [=identifier=] »
+     :  If the [{{Global}}] [=extended attribute=] [=takes an identifier list=]
+     :: the identifier list
+</dl>
+
+The [{{Global}}] extended attribute must be one of the forms given above.
+
+Note: The [=global names=] for the [=interface=] are the [=identifiers=] that can be used to
+reference it in the [{{Exposed}}] [=extended attribute=].
+A single name can be used to reference multiple global interfaces.
```
Comment (tabatkins) [inline index.bs]:
```suggestion
A single name can be shared across multiple different global interfaces, 
allowing an interface to more easily use [{{Exposed}}] to expose itself to all of them at once.
```

File: index.bs [hunk 2]
```diff
@@ -8673,12 +8673,33 @@ for the specific requirements that the use of
...
-interfaces. Specifically:
+it indicates that objects implementing this interface will
+be used as the global object in a [=Realm=].
+
+The [{{Global}}] [=extended attribute=] also defines the
+<dfn id="dfn-global-name" export>global names</dfn> for the [=interface=]:
+
+<dl class="switch">
+     :  If the [{{Global}}] [=extended attribute=] [=takes no arguments=]
+     :: « the [=identifier=] of the [=interface=] itself »
```
Comment (annevk) [inline index.bs:8684]: This was not there before, I don't think it's needed?

**Comment by domenic** [comment]
LGTM with same caveat.

File: index.bs (hunk 2 above)
Comment (Ms2ger) [inline index.bs:8684]:
```suggestion
```

File: index.bs [hunk 3]
```diff
@@ -9437,12 +9437,31 @@ for the specific requirements that the use of
...
+     :: the identifier list
+</dl>
+
+The [{{Global}}] extended attribute must be one of the forms given above.
+
+Note: The [=global names=] for the [=interface=] are the [=identifiers=] that can be used to
+reference it in the [{{Exposed}}] [=extended attribute=].
+A single name can be shared across multiple different global interfaces,
+allowing an interface to more easily use [{{Exposed}}] to expose itself to all of them at once.
+For example, "Worker" is used to refer to several distinct types of threading-related
```
Comment (TimothyGu) [inline index.bs:9459]:
```suggestion
For example, "<code>Worker</code>" is used to refer to several distinct types of threading-related
```

**Pull request merged**